adapters/rest-adapter-generator/src/main/java/uk/gov/justice/services/adapters/rest/generator/JaxRsImplementationGenerator.java

```diff
@@ -191,10 +188,8 @@ private FieldSpec dispatcherFieldFor(final Action action) {
 
         if (actionType == GET) {
```

Comment:
We think that this code was okay before with the if/else if/else throw exception as if we added a new supported type (e.g. PUT) then it could potentially fail silently.